### DISCUSSION TITLE
Add option to customise default settings for Tasks

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#337] [Suggestions]
Hey my idea is, that we have that setting from the following picture as a group setting (The setting for "Activate for ... Clusters and/or Servers

![image](https://user-images.githubusercontent.com/99832504/216546084-b709efd4-d989-4b91-9436-d009d0e2a34b.png) 

in 

![image](https://user-images.githubusercontent.com/99832504/216546345-51cf9455-7ae8-423b-8c98-b02fed9fb71c.png)

as a group setting like those for shop packs

![image](https://user-images.githubusercontent.com/99832504/216546580-e3e6db1b-108a-4909-863c-a5f8752ec260.png)

I hope that I explained it good :D

**Comment by mvaessen** [comment]
Hi, thanks for your sugegstion.
Do you mean you want to have a general place where you can setup the default settings for `achievement` tasks?

**Reply by LimitlessDrag0n** [reply]
Yes, that's the words I meant.
